### ISSUE TITLE
What is the loss of BERT and is it a muti-task training task?

### ISSUE THREAD

**Issue by nanguoshun** [RaleighZ/statnlp_fundamental_reading#8]

**Comment by RaleighZ** [comment]
I find a blog containing the details of [BERT](https://plmsmile.github.io/2018/12/15/52-bert/).
We can find in the blog that 
1. "the loss of training BERT is the sum of MLM loss and NSP loss (训练loss：LM和NSP的loss加起来)". 
2. "15% of tokens are randomly masked after wordpiece tokenization（WordPiece Tokenization后再mask掉15%的词汇。）”

**Comment by RaleighZ** [comment]
In BERT paper:
1. "The training loss is the sum of the mean masked LM likelihood and mean next sentence prediction likelihood."
2. "To generate each training input sequence, we **sample two spans** of text from the corpus, which we refer to as “sentences” even though they are typically much longer than single sentences (but can be shorter also). The first sentence receives the A embedding and the second receives the B embedding. 50% of the time B is the actual next sentence that follows A and 50% of the time it is a random sentence, which is done for the “next sentence prediction” task. They are sampled such that the combined length is no more than 512 tokens. **The LM masking is applied after WordPiece tokenization with a uniform masking rate of 15%, and no special consideration given to partial word pieces.**"
